x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/insights_tab_csp.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiButtonGroup, EuiSpacer } from '@elastic/eui';
+import type { EuiButtonGroupOptionProps } from '@elastic/eui/src/components/button/button_group/button_group';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useExpandableFlyoutState } from '@kbn/expandable-flyout';
+import { MisconfigurationFindingsDetailsTable } from './misconfiguration_findings_details_table';
+
+const insightsButtons: EuiButtonGroupOptionProps[] = [
+  {
+    id: 'misconfigurationTabId',
```

Comment:
let's add this id to a constant, so we avoid updating the id name and forgetting to update the `activeInsightsId`